<launch>
    <node pkg="bw_shared_config" type="shared_config_node.py" name="shared_config_node"
        output="screen" />

    <group ns="mini_bot">
        <node name="motor_characterize" pkg="bw_teleop" type="motor_characterize_node.py"
            output="screen" required="true">
            <param name="audio_directory" value="/media/storage/audio" />
            <param name="channel" value="1" />
            <param name="microphone_id" value="10" />
        </node>
        <include file="$(find bw_teleop)/launch/mini_bot_bridge.launch">
            <param name="lookup_table" value="passthrough" />
        </include>
        <node pkg="rosbag" type="record" name="rosbag_characterize_motors"
            args="record -o /media/storage/bags/characterize_motors microphone_sample" />
    </group>
</launch>